### PR TITLE
sets autoComplete to off in Dropdown #808

### PR DESCRIPTION
**Pull request by AnnMarieW**
Currently, the default for autoComplete is "on" in the Dropdown component.   When autoComplete="on", the browser will autocomplete fields based on the history stored in the browser.

image

This pull request addresses #808, plotly/dash#960 and #787 by setting the autoComplete default to "off"

This issue has also been mentioned numerous times on the community forum:
https://community.plotly.com/t/disable-autocomplete-for-dropdowns-input-box-in-dash/22463
https://community.plotly.com/t/disable-browser-autocomplete-on-multiselect-dropdown/37478
https://community.plotly.com/t/keep-getting-autofill-for-dropdown-not-all-browsers/30359

### CODE REVIEW

File: src/fragments/Dropdown.react.js [hunk 1]
```diff
@@ -97,6 +98,7 @@ export default class Dropdown extends Component {
                     onInputChange={search_value => setProps({search_value})}
                     backspaceRemoves={clearable}
                     deleteRemoves={clearable}
+                    inputProps={{...inputProps, autoComplete: 'off'}}
```
Comment (alexcjohnson) [inline src/fragments/Dropdown.react.js:101]: We're not making an `inputProps` prop anymore, right?

Comment (AnnMarieW) [reply src/fragments/Dropdown.react.js:101]: inputProps is not a dash prop -  it's a prop in the react component.  In order to update autoComplete, it looked like it needed to be wrapped in this way.  (it works :-))

Comment (AnnMarieW) [reply src/fragments/Dropdown.react.js:101]: Also I'm not sure why the tests failed.  I didn't make any changes to:
test_grva004_graph_prepend_trace[False]
test_grva005_graph_extend_trace[False]

Comment (alexcjohnson) [reply src/fragments/Dropdown.react.js:101]: Right, it's a prop of `ReactDropdown`, but what I meant was:
```suggestion
                    inputProps={{autoComplete: 'off'}}
```
ie you're pulling it in [up above](https://github.com/plotly/dash-core-components/pull/923/files#diff-6d73123aa77cd179dde9336ee11b3c81ba48c5589b504e8bb9e885a806a05473R57) as if it were a dash prop.

Comment (alexcjohnson) [reply src/fragments/Dropdown.react.js:101]: Not sure what's going on with those tests, but whenever a test fails in only one of the Python versions the first guess is it's just flaky. We'll see what happens when we try it again.

Comment (AnnMarieW) [reply src/fragments/Dropdown.react.js:101]: Oh, I see.  I'll make the change.

**Review by alexcjohnson — APPROVED** [review]
Super! Just need a changelog entry then this is ready to go! 💃

**Comment by BMJHayward** [comment]
Looking forward to this - any idea when it'll land in pip?

**Comment by harryturr** [comment]
@BMJHayward It was just released in [`dash-core-components==1.16.0`](https://pypi.org/project/dash-core-components/1.16.0/)!

**Comment by adrianoesch** [comment]
I was about to get to work on this, when i found this PR and merely had to upgrade dash. Thanks a lot! 😘

**Comment by frankyp1234** [comment]
Still having this issue - have there been any updates?

**Comment by AnnMarieW** [comment]
@frankyp1234 This issue was solved and closed.  What version of Dash are you using?  Can you make a minimal example that reproduces the issue?

**Comment by kailasvnair49** [comment]
I have the same issue in the Safari browser. working fine in the Chrome browser.

**Comment by ghost** [comment]
I have the same issue in safari.

**Pull request merged**